client/src/components/PageLayout/component/Footer.tsx

```diff
@@ -1,33 +1,44 @@
-import { Link, Flex } from '@chakra-ui/react';
+import { Link, Grid } from '@chakra-ui/react';
 import React from 'react';
 
 export const Footer: React.FC = () => {
   return (
-    <Flex
+    <Grid
       as={'footer'}
       marginTop="2em"
       backgroundColor={'gray.10'}
       paddingInline="2em"
       paddingBlock="1em"
-      gap="2em"
-      justifyContent="space-evenly"
+      gap="3em"
+      gridTemplateColumns="repeat(auto-fit, minmax(8rem, 1fr))"
```

Comment:
I am abusing this 😆

Reply:
Abuse away. It looks nice and it responsive, so it clearly works!